src/JuliaInterpreter.jl

```diff
@@ -25,7 +25,7 @@ const BUILTIN_FILE = joinpath(@__DIR__, "builtins-julia$(Int(VERSION.major)).$(I
 @info "Generating builtins for this julia version..."
 gen_builtins_file = joinpath(@__DIR__, "generate_builtins.jl")
 # Run as separate command to prevent including the generate_builtins into the precompile cache
-run(`$(Base.julia_cmd()) --startup-file=no $gen_builtins_file`)
+run(`$(Base.julia_cmd()) --project=@. --startup-file=no $gen_builtins_file`)
```

Comment:
```suggestion
run(`$(Base.julia_cmd()) --project=$(joinpath(@__DIR__, "..") --startup-file=no $gen_builtins_file`)
```

Reply:
Thanks. I wonder if all this is worth it just to save caching the builtin generator... Oh well.